packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -523,7 +525,7 @@ const ToggleMenuDrawer = observer(({ platform_config }) => {
                                 {is_logged_in && (
                                     <MobileDrawer.Item>
                                         <MenuLink
-                                            link_to={routes.traders_hub}
+                                            link_to={is_next_wallet_enabled ? routes.wallets : routes.traders_hub}
```

Comment:
This ensures that when the wallets feature flag is enabled, when user clicks on Trader's Hub in responsive, they are routed to `/wallets`:

https://github.com/binary-com/deriv-app/assets/103016120/e2767ec3-bda7-422a-a8d0-0351f044dd25